mungegithub/submit-queue/www/index.html

```diff
@@ -148,12 +148,21 @@ <h4 class="md-body-2">{{pr.Reason}}</h4>
               <md-toolbar class="md-whiteframe-z2">
                 <h2 class="md-toolbar-tools">End-to-End Results</h2>
               </md-toolbar>
+              <h2 class="md-title" ng-show="cntl.builds.length > 0">Queue-Blocking Builds</h2>
               <md-chips ng-model="cntl.builds" readonly="true">
```

Comment:
Spent some time on this and it sounds like it would work, but I couldn't get it working.  Can we refactor when needed in a follow-up PR?